plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/nessie/IcebergNessieCatalogModule.java

```diff
@@ -46,9 +46,9 @@ protected void setup(Binder binder)
     public static NessieIcebergClient createNessieIcebergClient(IcebergNessieCatalogConfig icebergNessieCatalogConfig)
     {
         return new NessieIcebergClient(
-                HttpClientBuilder.builder()
+                NessieClientBuilder.createClientBuilder(null, null)
```

Comment:
javadoc of `HttpClientBuilder.builder()` suggests migrating to `NessieClientBuilder#createClientBuilderFromSystemSettings()`. Why do we use `createClientBuilder()` method?